codebuild/fuzz_codebuild.config

```diff
@@ -14,7 +14,7 @@ buildspec: codebuild/spec/buildspec_ubuntu_fuzz_artifacts.yml
 source_location: https://github.com/awslabs/private-s2n-fuzz.git
 source_type : GITHUB
 source_clonedepth: 1
-source_version: master
+source_version:
```

Comment:
What would be the default here? Do we need this line still?

Reply:
From description: The documentation for source_version says: "If not specified, the default branch's HEAD commit ID is used." We leave source_version empty in other CodeBuild configs: https://github.com/awslabs/s2n/blob/master/codebuild/integ_codebuild.config#L29

Reply:
The documentation for source_version says: "If not specified, the default branch's HEAD commit ID is used." We leave source_version empty in other CodeBuild configs: https://github.com/awslabs/s2n/blob/master/codebuild/integ_codebuild.config#L29

The field does have to exist though or validation throws an error.